BrowserKit/Sources/UnifiedSearchKit/SearchEngineTableView.swift

```diff
@@ -0,0 +1,104 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+import Common
+import MenuKit
+
+// FXIOS-10189 This class will be refactored into a generic UITableView solution later. For now, it is largely a clone of
+// MenuKit's work. Eventually both this target and the MenuKit target will leverage a common reusable tableView component.
+public final class SearchEngineTableView: UIView,
+                             UITableViewDelegate,
+                             UITableViewDataSource, ThemeApplicable {
```

Comment:
this.... is weird. :P

Reply:
[I copied it from MenuTableView](https://github.com/mozilla-mobile/firefox-ios/blob/d82ae0028f13738136c22e7bded49d509f646b14/BrowserKit/Sources/MenuKit/MenuTableView.swift#L15)... 👀 

I figured there was a reason. My guess is when you make a newline for ThemeApplicable, the automatic xcode left alignment is off? 
<img width="492" alt="Screenshot 2024-11-06 at 2 46 16 PM" src="https://github.com/user-attachments/assets/b67f8d1a-2f9f-414d-b9d8-6194d278c7bd">


I also thought it strange. Want me to change both files to manually align both these cases?

Reply:
GASP! Did I do that? SHAME!

Yes plsz!

Reply:
and my comment is minor.

Reply:
Ok perfect, will do! 👌 😂